src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/RC2Implementation.OpenSsl.cs

```diff
@@ -41,11 +41,21 @@ private static OpenSslCipherLite CreateLiteCipher(
             return new OpenSslCipherLite(algorithm, blockSize, paddingSize, key, iv, encrypting);
         }
 
-        private static IntPtr GetAlgorithm(CipherMode cipherMode) => cipherMode switch
+        private static IntPtr GetAlgorithm(CipherMode cipherMode)
+        {
+            IntPtr algorithm = cipherMode switch
             {
                 CipherMode.CBC => Interop.Crypto.EvpRC2Cbc(),
                 CipherMode.ECB => Interop.Crypto.EvpRC2Ecb(),
                 _ => throw new NotSupportedException(),
```

Comment:
I wonder, are CFB and OFB intentionally not supported?

Reply:
Yes.  Windows only supports CFB8 with RC2, and OpenSSL only supports CFB64.  Since they had no intersection, RC2 in .NET doesn't support CFB.

I could be wrong, but I don't believe OFB is ever supported in .NET.  It's in the enum, but not handled by any type.

Reply:
Thanks!